ArcGISRuntimeSDKQt_CppSamples/Routing/NavigateRoute/NavigateRoute.cpp

```diff
@@ -270,7 +268,10 @@ void NavigateRoute::connectRouteTrackerSignals()
       // navigate to next stop, if available
       if (trackingStatus->remainingDestinationCount() > 1)
       {
-        m_routeTracker->switchToNextDestination();
+        m_routeTracker->switchToNextDestinationAsync().then(this, [this]()
+        {
+          m_routeTracker->generateVoiceGuidance();
```

Comment:
This doesn't look right. `m_routeTracker->generateVoiceGuidance();` was previously called when `RouteTracker::trackLocationCompleted` emitted. Now it's at the completion of `switchToNextDestinationAsync` so the logic is different.

Is this intentional?

Reply:
No. It is mistake while fixing some merge conflicts. Reverted.